lib/mosaic-geomath/src/main/java/org/eclipse/mosaic/lib/spatial/PointCloud.java

```diff
@@ -0,0 +1,317 @@
+/*
+ * Copyright (c) 2024 Fraunhofer FOKUS and others. All rights reserved.
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contact: mosaic@fokus.fraunhofer.de
+ */
+
+package org.eclipse.mosaic.lib.spatial;
```

Comment:
Looking at the other classes in the `spatial`-package This class doesn't quite fit in. Should we create a new sub-package, e.g., `detection`, or `perception`, or `imaging`, or `sensors`?

Reply:
I had a longer discussion with @fabmax on that. The best place actually would be `mosaic-objects`, but we didn't want PHABMACS to include this as an additional dependency. We decided, if we name it rather generalist (thus "point cloud" instead of "LidarFrame"), it could indeed fit into the "spatial" package, as it is just a set of points in the 3D space with a geo-reference. Generally speaking a point cloud must not be only the result of a sensor measurement.

Reply:
But I have to look if we could remove "minrange" and "maxrange", as this might be not used anywhere and should not be part of the "point cloud" as this is indeed sensor-specific properties.